src/main/resources/docs/mixing-assets.md

```diff
@@ -58,14 +61,55 @@ This comes in very handy during development, especially when you are tweaking yo
 
 >/alert/ <span> If you are serving your app, the recompilation of assets are not automatically
 >reflected in your browser. You have to recompile and reserve your app again.</span>
+</div>
+
+<a name="auto-reloading"></a>
+### [Watching Changes and Auto Reloading](#auto-reloading)
+
+<a name="auto-reload-frontend"></a>
+#### [Frontend Asset Changes](#auto-reload-frontend)
+
+When you are working with tweaking the design of your app or making the interactive aspect of the app just right, you
```

Comment:
`When you are tweaking the design of your app or making the interactive aspect of the app just right, you
may want your assets to recompile and the browser to reload automatically as soon as you save your changes.
Alpas comes wired with everything to facilitate this rapid prototyping so that you can iterate on your design faster
without having to recompile everything and refreshing your browser for every small change.`

Reply:
Thanks!